tonic/models/flo2d/utils.py

```diff
@@ -0,0 +1,140 @@
+from osgeo import osr
+import mpl_toolkits.basemap.pyproj as pyproj
+
+import numpy as np
+import pandas as pd
+
+from scipy import stats
+import quantities as pq
+from scipy.spatial import cKDTree
+
+# Default projection from Alan Hamlet 2015-06-29
+default_proj = pyproj.Proj('+proj=lcc +lat_1=47.5 +lat_2=48.73333333333333 '
+                           '+lat_0=47'' +lon_0=-120.8333333333333 '
+                           '+x_0=609601.2192024385 +y_0=0 +datum=NAD83 '
+                           '+units=us-ft')
+
+
+def esriprj2standards(shapeprj_path, kind='proj4'):
+    '''Return string representation of projection in shapeprj_path.
+
+    Parameters
+    ----------
+    shapeprj_path : str
+        Filepath to ESRI style projection file.
+    kind : str {'prj', 'wkt', 'proj4', 'epsg'}
+        Projection standard.
+
+    Returns
+    ----------
+    projection_params : str
+        Projection parameters.
+    '''
+    prj_file = open(shapeprj_path, 'r')
+    prj_txt = prj_file.read()
+    srs = osr.SpatialReference()
+    srs.ImportFromESRI([prj_txt])
+    if kind.lower() == 'prj':
+        return prj_txt
+    if kind.lower() == 'wkt':
+        return srs.ExportToWkt()
+    elif kind.lower() == 'proj4':
+        return srs.ExportToProj4()
+    elif kind.lower() == 'epsg':
+        srs.AutoIdentifyEPSG()
+        return srs.GetAuthorityCode(None)
+    else:
+        raise ValueError('Kind is unknown')
+
+
+def read_flo2d_depth_file(fname, names=('x', 'y', 'depth')):
+    '''Parser helper function to read ASCII output from FLO-2D
+
+    Parameters
+    ----------
+    fname : str
+        Filepath to FLO-2D depth file.
+    names : tuple of str, optional
+        Header names
+
+    Returns
+    ----------
+    df : Pandas.DataFrame
+        DataFrame of FLO-2D output.
+    '''
+    return pd.read_table(fname, sep='\s+',
+                         names=names,
+                         index_col=0)
+
+
+def make_coordinates(arr):
```

Comment:
make variable more informative, like instead of `arr` `coordinate_array`?

or perhaps this is a question of best practices and you save key strokes by just doing `arr` and your explanation of what kind of `arr` it is in the description of the function